src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -62,6 +62,9 @@
         "organism_age_unit", "TerraCore:hasAgeUnit"
     );
 
+    private final Map<String, String> fieldReplacements =
```

Comment:
I wonder if I should replace this with a BiMap. It would make traversing the object easy!

Reply:
I replaced this with a bimap, which simplified the code a bit.